src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,33 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private double bucketCapacity;
```

Comment:
Access modifiers for fields should be private to adhere to encapsulation. In this case, it is correct, but ensure that this is consistent across all model classes.